### DISCUSSION TITLE
The introduction to autonomous flight in the official documentation is so scarce that we can hardly continue the study

### DISCUSSION THREAD

**Discussion by wtmzwelkin** [bitcraze/discussions#925] [Q&A]
The introduction to autonomous flight in the official documentation is so scarce that we can hardly continue the study
![屏幕截图 2023-09-22 160531](https://github.com/bitcraze/discussions/assets/121012440/6578530d-79df-4eea-822e-905a7824525a)

**Comment by knmcguire** [comment]
Hi! Maybe the step-by-step guides of the python library will be more useful for you? You can start there and then you should know enough to get started with the other example in that directory

https://www.bitcraze.io/documentation/tutorials/#software
